worlds/cv64/docs/en_Castlevania 64.md

```diff
@@ -116,7 +116,7 @@ Enabling Carrie Logic will also expect the following:
 
 - Orb-sniping dogs through the front gates in Villa
 
-Library Skip is **NOT** logically expected on any setting. The basement hallway crack will always logically expect two Nitros
+Library Skip is **NOT** logically expected with any option. The basement arena crack will always logically expect two Nitros
```

Comment:
The "with any option" here and in `options.py` are tripping me up a bit. I feel like "by any options" and "under any options" are both a bit better. Alternatively, you could just say "X is/are **NEVER** considered in logic"

Reply:
Changed it to your first suggestion.